graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -174,3 +180,33 @@ def handle_driver_exception(session: neo4j.Session, e: Exception) -> None:
         all_endpoints = list_result.to_df()["name"].tolist()
 
         raise SyntaxError(generate_suggestive_error_message(requested_endpoint, all_endpoints)) from e
+
+    def _verify_connectivity(self) -> None:
+        WAIT_TIME = 1
+        MAX_RETRYS = 10 * 60
```

Comment:
wonder if we want to allow users to configure this timeout

Reply:
Yes, I thought about that too. My reasoning for not doing it, was to wait for feedback to see if anyone actually wants it. Don't want to complicate the surface needlessly. But it would be very easy to add of course. Wdyt?

Reply:
Waiting for feedback sounds good. We could also ask Dave directly what his opinion is (or the gds power users).

Reply:
Let's wait then :+1: